templates/oned.conf.erb

```diff
@@ -340,6 +376,10 @@ IM_MAD = [
 
 #-------------------------------------------------------------------------------
 #  EC2 Information Driver Manager Configuration
+<% if scope.function_versioncmp([@one_version,'4.14']) >= 0 -%>
+#    -r number of retries when monitoring a host
+#    -t number of threads, i.e. number of hosts monitored at the same time
+<% end -%>
```

Comment:
Same as above: I would keep the documentation inside. Are these parameters only valid in 4.14? In this case just add a comment which versions make use of the parameters.